lib/drivers/ryo-explorer.js

```diff
@@ -15,7 +15,7 @@ class RyoExplorer extends Driver {
         circulating: true,
         total: true,
         max: true,
-        blockchains: ['Ryo'],
+        blockchains: ['Cryptonote'],
```

Comment:
I'd keep this as `Ryo` because `CryptoNote` is more of a protocol than a blockchain. If your blockchain is based on the CryptoNote protocol, you'd still have to launch your own blockchain.